File: 3379-score-of-a-string/3379-score-of-a-string.java
class Solution {
    public int scoreOfString(String s) {
        int left = 0 ,right = 1;
        int score = 0;
        int len = s.length();
        StringBuilder sb = new StringBuilder(s);
        System.out.println((int)s.charAt(0));
        while (right < len){
           score += Math.abs(sb.charAt(left++) - sb.charAt(right++));
        }
        return score;
    }
}

File: 3379-score-of-a-string/3379-score-of-a-string.cs
public class Solution {
    public int ScoreOfString(string s) {
        int ans = 0;
        for(int i = 0; i < s.Length-1; i++){
            ans += Math.Abs((s[i] - 'a') - (s[i+1] - 'a'));
        }
        return ans;
    }
}